.eslintrc.json

```diff
@@ -13,28 +13,26 @@
 		"plugin:promise/recommended",
 		"plugin:jest/recommended",
 		"plugin:jest/style",
-		"prettier",
-		"prettier/@typescript-eslint"
+		"prettier"
 	],
 	"rules": {
 		"import/prefer-default-export": "off",
 		"import/no-default-export": "error",
-		"tsdoc/syntax": "error",
 		"no-restricted-syntax": [
 			"error",
 			"ForInStatement",
 			"LabeledStatement",
 			"WithStatement"
 		],
 		"no-return-assign": ["error", "except-parens"],
-		"github/array-foreach": ["error"],
+		"unicorn/no-array-for-each": "error",
```

Comment:
Why did you switch from GitHub to unicorn?

Reply:
See the commit log. This is the same type of rule as `github/array-foreach`, but it's auto-fixable.

Also, unicorn has a lot of other interesting rules we may want to use in the future (not enabled now cause I don't want to break existing stuff).